service/service.go

```diff
@@ -54,6 +54,7 @@ type authGuard interface {
 
 type settingsGetter interface {
 	GetGithubSettings(ctx context.Context) (model.GithubSettings, error)
+	GetSlackSettings(ctx context.Context) (settings model.SlackSettings, err error)
```

Comment:
As suggested in #37, it might be better to have `GetSlackToken(ctx context.Context) (string, error)`.